### PR TITLE
Feat/allow more add styling

### PR DESCRIPTION
**Pull request by malteludwig**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation content changes

## Other information

**adds more vars for add button styling**

Issue Number: N/A

### CODE REVIEW

File: src/components/plugins/add/add.scss [hunk 1]
```diff
@@ -20,20 +20,59 @@ button {
...
+   * @default 0.2rem
+   */
   color: var(--stylo-add-color, var(--medium));
 
   &:hover,
   &:focus {
-    color: var(--light-contrast);
+    /**
+   * @prop --stylo-add-color-hover: The hover color of the button
+   * @default 0.2rem
```
Comment (peterpeterparker) [inline src/components/plugins/add/add.scss:63]: I'll correct few comments that have incorrect default values after merge

**Review by peterpeterparker — APPROVED** [review]
Great, thanks!

**Comment by malteludwig** [comment]
Thanks for correcting and merging! 

I have a couple of other things in mind I’d like to work on next week. 

I think it would be nice to have optional display of plugin labels and a Vertical display of the plugin icon list.

Also I’ll think of a Vimeo/Yt  plugin 

let me know.

You got a very cool lib there mate, good work!

**Comment by peterpeterparker** [comment]
> optional display of plugin labels

Not against as long as the actions remain accessible

> Vertical display of the plugin icon list

Not sure there is a need but if it fits yours, sure go on

> Also I’ll think of a Vimeo/Yt plugin

I am not keen to add additional plugins to this repo because it means I would have to maintain these.
Developers using Stylo can develop and configure their own plugins.
If you develop some, not against listing these in a "Community Plugins" chapter of the README.

Thanks for the ideas and PRs 👍

**Pull request merged**